README.md

```diff
@@ -1,12 +1,21 @@
 # .NET Core Home
 
 The core repository is the starting point to engage in and learn about the
-.NET Core stack.
+.NET Core stack. 
+
+> **Note:** please note that this repository is not for filing product issues.
+> If you run into an issue using .NET Core, there are multiple repos where you can 
+> file an issue:
+> * dotnet/cli - for CLI tools and questions
+> * dotnet/corefx - for API issues and questions
+> * dotnet/coreclr - for runtime issues
+> * nuget/home - for NuGet questions and issues
+> We will be removing the ability to file issues on this repo in the near future.
```

Comment:
* Where should I file an issue if it does not fall clearly within one of these categories or if I'm unsure where it belongs?
* These should be links.
* Also mention aspnet/Home?